Gentoo bug     : https://bugs.gentoo.org/show_bug.cgi?id=424882
Upstream bug   : http://www.thewildbeast.co.uk/claws-mail/bugzilla/show_bug.cgi?id=2689
Upstream commit: http://www.claws-mail.org/tracker/getpatchset.php?ver=3.8.1cvs12

Claws Mail segfaults showing GPG/S-MIME signature info.


Index: src/plugins/pgpcore/sgpgme.c
===================================================================
RCS file: //claws/src/plugins/pgpcore/Attic/sgpgme.c,v
retrieving revision 1.1.2.71
retrieving revision 1.1.2.72
diff -u -r1.1.2.71 -r1.1.2.72
--- src/plugins/pgpcore/sgpgme.c	8 Jul 2012 12:48:47 -0000	1.1.2.71
+++ src/plugins/pgpcore/sgpgme.c	10 Jul 2012 14:10:11 -0000	1.1.2.72
@@ -327,7 +327,7 @@
 				j++;
 				user = user->next;
 			}
-			g_string_append_printf(siginfo,
+			g_string_append(siginfo,
 				_("Primary key fingerprint:"));
 			const char* primary_fpr = NULL;
 			if (key && key->subkeys && key->subkeys->fpr)

